Assignment/Assignment.Tests/PingProcessTests.cs

```diff
@@ -87,36 +93,52 @@ async public Task RunAsync_UsingTpl_Success()
     [ExpectedException(typeof(AggregateException))]
     public void RunAsync_UsingTplWithCancellation_CatchAggregateExceptionWrapping()
     {
-        
+        CancellationTokenSource cts = new CancellationTokenSource();

```

Comment:
This can be simplified:
```suggestion
        CancellationTokenSource cts = new();

```